app/com/yetu/play/authenticator/messages/LogoutEventMsg.scala

```diff
@@ -0,0 +1,20 @@
+package com.yetu.play.authenticator.messages
+
+import play.api.data.validation.ValidationError
+import play.api.libs.json.{JsPath, Json}
+
+case class LogoutEventMsg(userId: String)
+
+object LogoutEventMsg {
+
+  val event = "logout"
+
+  implicit val logout_event_msg = Json.format[LogoutEventMsg]
+}
+
+
+case object UsedDeletedMsg
+
+case object EventNotMatchedMsg
```

Comment:
can we add a `sealed trait EventMessage` and have all of these case classes and case objects extend that trait? That will then give a compiler warning if you match on any message and you forgot to match on one of them.